src/lib-components/colors.ts

```diff
@@ -36,26 +46,42 @@ export function hslToHex(hParam: number, sParam: number, lParam: number): string
     g = hue2rgb(p, q, h);
     b = hue2rgb(p, q, h - 1 / 3);
   }
-  const toHex = (x: number): string => {
-    const hex = Math.round(x * 255).toString(16);
-    return hex.length === 1 ? `0${hex}` : hex;
-  };
   return `#${toHex(r)}${toHex(g)}${toHex(b)}`;
 }
 
+export function cubeHelix(h: number, s: number, l: number) {
+  const A = -0.14861;
+  const B = +1.78277;
+  const C = -0.29227;
+  const D = -0.90649;
+  const E = +1.97294;
+  const hRad = (h + 120) * (Math.PI / 180);
+  const a = s * l * (1 - l);
+  const cosh = Math.cos(hRad);
+  const sinh = Math.sin(hRad);
```

Comment:
`sinh` and `cosh` usually mean hyperbolic functions <https://en.wikipedia.org/wiki/Hyperbolic_functions>, but OK :)

Reply:
possible replacement: `sinHue`, `cosHue` if you prefer that.